app/src/entities/compactMessage/api/searchCompactMessages.ts

```diff
@@ -0,0 +1,18 @@
+import { compactMessagesMock } from '@entities/compactMessage/mock/compactMessagesMock';
+import { CompactMessage } from '@entities/compactMessage/model/compactMessage';
+
+import { getPaginateArray } from '@shared/lib/utils/getPaginateArray';
+
+export const searchCompactMessages = async (
+    substring: string,
+    page: number = 1,
+    maxSize: number = 15,
+): Promise<CompactMessage[]> => {
+    const array = compactMessagesMock.filter((compactMessage: CompactMessage) =>
```

Comment:
```suggestion
    const array = compactMessagesMock.filter((compactMessage) =>
```
если всё ок, иде тянет тип из списка